source/features/add-pr-branches.js

```diff
@@ -0,0 +1,54 @@
+import { h } from 'dom-chef';
+import select from 'select-dom';
+import * as cache from '../libs/cache';
+import * as api from '../libs/api';
+import { getOwnerAndRepo } from '../libs/page-detect';
+
+async function fetchFromApi(owner, repo, number) {
+	const response = await api.v3(`repos/${owner}/${repo}/pulls/${number}`); // https://developer.github.com/v3/pulls/#get-a-single-pull-request --> /repos/:owner/:repo/pulls/:number
+	if (response && response.base && response.head) {
+		return {
+			base: {
+				label: response.base.label,
+				url: response.base.repo.html_url + '/tree/' + response.base.ref
+			},
+			head: {
+				label: response.head.label,
+				url: response.head.repo.html_url + '/tree/' + response.head.ref
```

Comment:
Use template literals here.

```suggestion
				url: `${response.head.repo.html_url}/tree/${response.head.ref}`
```